lib/config/migration.spec.ts

```diff
@@ -133,6 +136,14 @@ describe('config/migration', () => {
           automerge: 'minor',
           schedule: null,
         },
+        python: {
+          packageRules: [
+            {
+              matchPackageNames: ['foo'],
```

Comment:
What should happen if the nested `packageRule` already contains a `matchCategories` field?
Merge or overwrite? 
The second one is the case right now.

Reply:
I think overwrite is fine in such an extreme edge case